test/test_heuristics.rb

```diff
@@ -147,7 +147,7 @@ def test_obj_c_by_heuristics
     # Only calling out '.h' filenames as these are the ones causing issues
     assert_heuristics({
       "Objective-C" => all_fixtures("Objective-C", "*.h"),
-      "C++" => ["C++/scanner.h", "C++/qscicommand.h", "C++/v8.h", "C++/gdsdbreader.h"],
+      "C++" => ["C++/scanner.h", "C++/v8.h", "C++/gdsdbreader.h"],
```

Comment:
Could you replace `C++/qscicommand.h` with `C++/protocol-buffer.pb.h` instead of removing it?